### ISSUE TITLE
Simple camera take picture component

### ISSUE THREAD

**Issue by art0007i** [Yellow-Dog-Man/Resonite-Issues#515]
### Is your feature request related to a problem? Please describe.

I want to make a simple camera that saves a photo to the screenshots folder, and generates metadata for that photo.

### Describe the solution you'd like

A component with a delegate or a protoflux node which will work similar to the "render to texture asset" flux node but it will also create a photo item under a specified slot, with the PhotoMetadata component and then save that photo to the screenshots folder like an interactive camera does.

### Describe alternatives you've considered

Using the interactive camera component, but it has a lot of stuff I don't need and don't want (like a secondary camera)
Additionally the interactive camera has some issues with taking pictures when it is inside of userspace.

### Additional Context

I specifically want this for a facet, so I'd prefer if the metadata is related to the world I'm in, not the userspace world.

**Comment by Frooxius** [comment]
This is something we'd need to approach cautiously, because this lets you put stuff into user's folders without their consent, which could be abused.

We would need some kind of mechanism to ensure that this cannot be abused, but that increases the engineering complexity of this.

**Comment by art0007i** [comment]
I think if the interactive camera component can exist then so can this. 

I just want a simplified interactive camera that only takes pictures and doesn't do anything fancy like the actual camera.

**Comment by Frooxius** [comment]
I'm not saying it can't exist. I am saying that we will need to consider the implications of this.

I understand why you want this feature, but that does not make us not consider the implications of adding this.

**Comment by TisFoolish** [comment]
I think Art's argument is that you can already force people to take photos via flux and the interactive camera component.

**Comment by Frooxius** [comment]
Yes, but this is a lot more direct way to do it, which warrants reconsideration on how we'll approach this kind of functionality.
